rust_src/crates/webrender/src/event_loop.rs

```diff
@@ -1,5 +1,15 @@
 use std::{cell::RefCell, os::unix::prelude::AsRawFd, ptr, sync::Mutex, time::Instant};
 
+#[cfg(macos)]
+use copypasta::osx_clipboard::OSXClipboardContext;
+#[cfg(windows)]
```

Comment:
I suppose you just add this for completeness but haven't actually tested it on windows ?